en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,54 @@
+Customize Content Attribute Display
+===================================
+
+Context
+-------
+
+In order to have minimum constraints on contents, Open Orchestra allows you to store any type
+of ``value`` in the ``ContentAttribute``.
+Sometimes, for instance in data tables, there is no easy way to render the value to the backoffice user so we have added
+a ``stringValue`` to ``ContentAttribute``.
+The ``stringValue`` is a projection of the ``value`` in a string.
+
+Transformers
+------------
+
+Adding a transformer
+~~~~~~~~~~~~~~~~~~~~
+
+Open Orchestra transforms the value into a string thanks to strategy implementing the interface
+``OpenOrchestra\Backoffice\ValueTransformer\ValueTransformerInterface``.
+Different strategies are already available in ``OpenOrchestra\Backoffice\ValueTransformer\Strategies``.
+For the service to be detected as a strategy, it should use the tag
+``open_orchestra_backoffice.value_transformer.strategy``:
+
+.. code-block:: yml
+
+    open_orchestra_backoffice.value_transformer.array:
+            class: %open_orchestra_backoffice.value_transformer.array.class%
+            tags:
+                - { name: open_orchestra_backoffice.value_transformer.strategy }
+
+Then implement the required methods:
+
+- ``transform``: transform value to an HTML string
+- ``support``: return true if the transformer must be used
+- ``getName``: uses by manager as index for the transformer
+
+Override a strategy
```

Comment:
you should be consistent with the other title : 
`adding a transformer` and `Override a strategy` are not